setup.py

```diff
@@ -39,6 +39,7 @@
                                                'tests.*', 'tests')),
     install_requires=(
         'requests>=2.18.4',
-        'tdigest>=0.5.2'
+        'tdigest>=0.5.2',
+        'cython>=0.29.7'
```

Comment:
Since `cython` isn't actually required to run the sdk, and there was a problem only in AWS lambda environments, we should describe the workaround for AWS lambda in README instead of adding this dependency here.